cmd/worker/main.go

```diff
@@ -35,7 +36,27 @@ const (
 	localPkgPathFmt = "/local/%s"
 )
 
-func handleMessage(ctx context.Context, msg *pubsub.Message, packagesBucket *blob.Bucket, resultsBucket, fileWritesBucket, imageTag string) error {
+type notificationMessage struct {
+	Package	resultstore.PkgIdentifier
+}
+
+func publishNotification(ctx context.Context, notificationTopic *pubsub.Topic, name, version, ecosystem string) error {
```

Comment:
See comment to `notificationMessage` - this function should go into the same package as that type